pkg/apis/configuration/v1alpha1/kong_license_types.go

```diff
@@ -0,0 +1,107 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Namespaced,shortName=kl,categories=kong-ingress-controller,path=konglicenses
+// +kubebuilder:storageversion
+// +kubebuilder:subresource:status
+
+// KongLicense stores a Kong enterprise license to apply to managed Kong gateway instances.
+type KongLicense struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	// RawLicenseString is the raw content of the license in string format.
```

Comment:
```suggestion
	// RawLicenseString is a string with the raw content of the license.
```
WDYT? I propose this change because in the original what does it mean "string format"